### ISSUE TITLE
integrating Tensorflow models flexibly

### ISSUE THREAD

**Issue by kdarras** [ecomontec/ecoSound-web#269]
We need to code the Tensorflow integration sustainably to flexibly integrate different kinds of TF models that will typically use an audio recording as input as well as optional parameters. These models will output time coordinates, labels, and confidence scores.

So we need a table for models, it could contain: tf_model_id, name, tf_model_path, labels_path, source_URL

We want to implement the following neural network first:

- BirdNET (Analyzer) https://github.com/kahst/BirdNET-Analyzer is a TensorFlow Lite-based neural net for automated detection of bird calls. This model produces more accurate predictions if the coordinates (present in the site table) and the week number (can be obtained from recording start time, after conversion of date_time to number of weeks) are supplied.

For each selected model, the corresponding optional parameters (coordinates and weeks number) should be extracted from user input or from the database.

**Comment by kdarras** [comment]
excellent user interface implementation. Working in Docker

**Comment by kdarras** [comment]
working in Docker
